requirements/edx/kernel.in

```diff
@@ -118,7 +118,7 @@ openedx-events>=3.1.0               # Open edX Events from Hooks Extension Frame
 openedx-filters                     # Open edX Filters from Hooks Extension Framework (OEP-50)
 # FIXME Use pip package once created
 # ref https://github.com/openedx/openedx-learning/pull/62
-git+https://github.com/open-craft/openedx-learning.git@jill/smarter-object-tags-rebase#egg=openedx-learning==0.1
+git+https://github.com/open-craft/openedx-learning.git@chris/system-defined-taxonomies#egg=openedx-learning
```

Comment:
TODO: merge and tag https://github.com/openedx/openedx-learning/pull/67, then update this dependency.